webapps/api/backend/views/vdisks.py

```diff
@@ -217,6 +217,27 @@ def move(self, vdisk, target_storagerouter_guid, force=False):
                                           target_storagerouter_guid=target_storagerouter_guid,
                                           force=force)
 
+    @action()
+    @log()
+    @required_roles(['read', 'write'])
+    @return_task()
+    @load(VDisk)
+    def move_multiple(self, vdisk, target_storagerouter_guid, force=False):
```

Comment:
This indicates the API would be on /api/vdisks/<vdisk_guid>/move_multiple which is not what I'd expect.
Due to our Django not supporting generic calls atm (https://github.com/openvstorage/framework/issues/1835), I'd opt to put it on the vpool level